platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/recovery/internal/RecoveryPlatform.java

```diff
@@ -88,7 +88,7 @@ public RecoveryPlatform(
 
         notificationEngine = NotificationEngine.buildEngine(getStaticThreadManager());
 
-        context = new DefaultPlatformContext(configuration, metrics, CryptographyHolder.get(), Time.getCurrent());
+        context = new DefaultPlatformContext(configuration, metrics, CryptographyHolder.get(), Time.getCurrent(), null);
```

Comment:
Should this be null? I think it would be ok to instantiate a file system manager in the current working directory for this.